app/directives/proposals/proposals.directive.html

```diff
@@ -1,3 +1,39 @@
 <div>
-    {{ctrl.proposals}}
+    <div class="col-xs-12 prop-header">
+        <div class="prop-row">
```

Comment:
This is header and flex is needed to set text and button at same line. Influence can be seen better in small devices. Without flex button is on the top of parent space.